content/conditions/dehydration/main-content.md

```diff
@@ -0,0 +1,96 @@
+!!! warning
+  Babies, children and the elderly are more at risk of dehydration. 
+!!!
+
+## Check if you’re dehydrated
+
+Symptoms of dehydration in adults and children include: 
+
+- feeling thirsty
+- dark yellow and strong smelling pee 
+- feeling dizzy or lightheaded 
+- feeling tired
+- dry mouth, lips and eyes
+- peeing little and less than 4 times a day
+
+If you have [diabetes](https://beta.nhs.uk/conditions/type-2-diabetes/check-if-you-have-it) or alcoholism
+you can dehydrate much more easily. 
+
+
+## How you can treat dehydration yourself
+
+Drink fluids when you feel any dehydration symptoms. Keep taking small sips. You can use a spoon to make it easier
+for your child to swallow the fluids.
+
+When you’re dehydrated your body also loses salt and sugar. To help restore sugar and salt levels you can drink: 
+
+- water
+- fruit juice or squash mixed with water
+- salty things like soup 
+
+Make sure you give your child a mix of these things so they can get back the sugars, salts or minerals they have lost. 
+
+
+## Your pharmacist can help with dehydration
+
+You can buy special sachets from a pharmacy. They’re called oral rehydration solutions and you mix them with water.
+They contain salt, sugar and minerals that your body loses when you’re not drinking enough. 
+
+Ask your pharmacist which ones are right for you or your child. 
+
+!!! info_compact
+  [Find your nearest pharmacy]({{ findersBaseUrl }}finders/find-help) 
+!!!
+
+!!! attention
+  See a GP if your symptoms don’t improve with treatment. 
+!!!
+
+!!! severe
+  ## Call 999 or go to A&E if  
+  
+  - you’re feeling unusually tired
+  - you’re confused and disorientated 
+  - any dizziness when you stand up doesn’t go away
+  - you haven’t peed for 8 hours
+  - your pulse is weak or rapid
+  - you have fits (seizures)
+  
+  ***
+  These can be signs of serious dehydration which needs urgent treatment.
+!!!
+
+
+## How to prevent dehydration
+
+Drink enough during the day so that your pee is a pale clear colour.
+
+Drink when you feel thirsty or when there is a higher risk of dehydrating. For example, if you’re vomiting, sweating or you have diarrhoea. 
+
+### Carers: making sure someone drinks enough
+
+Sometimes people you care for don’t have a sense of how much they’re drinking.
+
+To help them: 
+
+- make sure they drink during meal times
+- make drinking a social thing like ‘having a cup of tea’
+- offer them food with a high water content - for example, soups, ice cream, jellies or fruits like melon
+
+
+## Babies with dehydration
+
+!!! alert
+  ## Take your baby to the GP urgently or go to A&E if they
```

Comment:
The previous use of this has been "to the GP now" rather than "urgently". We should check with content as to whether this warrants a different use or whether we should be consistent.